mongoengine_migrate/fields/converters/common.py

```diff
@@ -379,15 +379,12 @@ def by_path(ctx: ByPathContext):
 
 def to_email_string(updater: DocumentUpdater):
     def by_path(ctx: ByPathContext):
-        fltr = {ctx.filter_dotpath: {'$not': email_regex, '$ne': None}, **ctx.extra_filter}
+        email_regex = r"\A[^\W][A-Z0-9._%+-]+@[A-Z0-9.-]+\.\p{L}+\Z"
```

Comment:
@vmdhhh A-Z in domain name (after @) should also be changed to \p{L} to support IDN domains.